python/guide/packages/gluon/custom-loss/custom-loss.md

```diff
@@ -1,11 +1,11 @@
 
-# Custom Loss functions in Gluon
+# Custom Loss Blocks
```

Comment:
Blocks for Custom Loss would be better wording

Reply:
Could make it harder to find in the navigation I think. Starting with Custom Loss cuts to the chase.